### DISCUSSION TITLE
Tooltip with different value for overflow-x and y.

### DISCUSSION THREAD

**Discussion by gcattan** [carbon-design-system/carbon#13881] [Help] [answered]
Hello, 

I have this problem, with the tooltip being hidden horizontally when the overflow-y of the parent div is set to 'scroll' or 'auto'.
https://codesandbox.io/s/carbon-react-tooltip-forked-dyfby4?file=/src/index.js

Do you know a possible workaround?

**Comment by tay1orjones** [comment]
Sorry for the delayed response. We're considering a large systemic fix for issues like this across the entire system in #14139

**Reply by gcattan** [reply]
No worries. Thank you very much for your answer.

**Comment by tay1orjones** [comment, marked as answer]
This was fixed in https://github.com/carbon-design-system/carbon/pull/14654, [here's an example ](https://react.carbondesignsystem.com/?path=/story/components-tooltip--experimental-auto-align)

**Comment by gcattan** [comment]
Thank @tay1orjones 
@fbarroso24 FYI
